index.html

```diff
@@ -871,6 +892,32 @@ <h3>Presentations</h3>
 
     </section>
 
+    <section class="informative">
+      <h3>Bound Credentials and Unbound Credentials</h3>
+
+      <p>
+        There are 2 broad categories of <a>verifiable credentials</a>: <a>bound credentials</a> and <a>unbound credentials</a>. 
+        <a>Bound credentials</a> are bound a specific credential <a>subject</a>. 
+        <a>Unbound credentials</a> are not bound to a <a>subject</a>. 
+        As such, <a>unbound credentials</a> cannot be used to create <a>verifiable presentations</a> -
+        they are simply "data that has been signed by an <a>Issuer</a> and held by any <a>Holder</a>".
+        <a>Unbound credentials</a> are easily issued to and exchanged between groups of two or more <a>Holders</a>. 
+        The most common use case for <a>unbound credentials</a> is to represent all types of business documents like invoices and purchase orders 
+        (that are not simple extensions or representations of a single subject).
+      </p>
+      <p>
+        <a>Bound credentials</a> contain a <a>subject</a> id element inside the credentialSubject element 
+      (which binds the credential to a subject and enables the <a>Holder</a> of the <a>credentential</a> 
+      to derive <a>verifiable presentations</a> from the <a>bound credential</a>).
+      </p>
+      <p>
+        <a>Unbound credentials</a> do not contain a <a>subject</a> id element inside the credentialSubject element.
+        An <a>unbound credential</a> is "like signed data issued to a <a>Holder</a>". 
```

Comment:
I don't know what this sentence means. When describing concepts, we should avoid using quotes that could be misinterpreted.

Reply:
> I don't know what this sentence means. When describing concepts, we should avoid using quotes that could be misinterpreted.

Don't get lost in the prose (yes, I realize this is a PR to the spec). As I indicated above (or is it below), I see a major restructuring of the credential spec and there's at least of few of us interested in tackling it now.  To reduce confusion, I'm going to call this new characterization Structured Credentials (a term I've already started to use and have an implementation of. Reference: https://www.youtube.com/watch?v=KXhyvKXCLus&list=PLU-rWqHm5p45dzXF2LJZjuNVJrOUR6DaD&index=1).